authorizations/authorization-ranger/src/main/java/org/apache/gravitino/authorization/ranger/RangerAuthorizationPlugin.java

```diff
@@ -745,6 +763,110 @@ public Boolean onGroupAcquired(Group group) {
     return Boolean.TRUE;
   }
 
+  @Override
+  public void close() throws IOException {
+    if (!isCreatedByPlugin) {
+      return;
+    }
+
+    try {
+      rangerClient.deleteService(rangerServiceName);
+    } catch (RangerServiceException rse) {
+      throw new AuthorizationPluginException(
+          "Fail to delete Ranger service %s, exception: %s", rangerServiceName, rse.getMessage());
+    }
```

Comment:
Why do we need the automated delete Reanger service?
Is the `RangerAuthorizationProperties.RANGER_SERVICE_CREATE_IF_ABSENT` only used by ITs?

Reply:
No. It's not only used by UT.
If we have a Ranger server and  Ranger HDFS or Hive services are not pre-created , we need the to create the by our Gravitino.
If these Ranger Hive services or HDFS services are created by Gravitino, Gravitino should delete them, too.